{"n_dim": 89, "n_trials": 340, "objective": "markowitz_skew_on_cube", "white": "bobyqa_noise", "black": "dlib_default", "traceback": ["passing", "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/humpday/comparison/eloratings.py\", line 76, in optimizer_game\n    black_best_val, black_best_x, black_feval_count = black(objective, n_trials=n_black_trials,\n  File \"/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/humpday/optimizers/dlibcube.py\", line 27, in dlib_default_cube\n    best_x, best_val = find_min_global(_objective, lb, ub, n_trials)\nRuntimeError: \n\nError detected at line 63.\nError detected in file /tmp/pip-install-icsp0ccd/dlib_0390dc858e4b4792b24184fe38745d92/tools/python/src/global_optimization.cpp.\nError detected in function double call_func(pybind11::object, const dlib::matrix<double, 0, 1>&).\n\nFailing expression was 0 < num && num < 35.\nFunctions being optimized must take between 1 and 35 scalar arguments.\n\n"], "best_val": [-0.055385113703867284, null], "best_x": [[0.08746961633908347, 0.44100607707656503, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.032251372202254214, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.08166034456396339, 0.0, 0.35680519057019333, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], null], "feval_count": [340, null], "n_trials_instructed": [340, 340], "passing": [true, false], "completed": false, "winner": "incomplete", "points": null}